packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/form/ViewFormDescriptionConverterSwitch.java

```diff
@@ -640,7 +640,13 @@ private String getItemId(VariableManager variableManager) {
 
     private Function<VariableManager, List<?>> getMultiValueProvider(String expression) {
         String safeExpression = Optional.ofNullable(expression).orElse("");
-        return variableManager -> this.interpreter.evaluateExpression(variableManager.getVariables(), safeExpression).asObjects().orElse(List.of());
+        return variableManager -> {
+            if (safeExpression.isBlank()) {
+                return List.of();
+            } else {
+                return this.interpreter.evaluateExpression(variableManager.getVariables(), safeExpression).asObjects().orElse(List.of());
+            }
+        };
     }
 
     private <T> Function<VariableManager, List<T>> getMultiValueProvider(String expression, Class<T> type) {
```

Comment:
Could you also applied the same change to this method? It will handle PieChart/BarChart who suffer the same problem.

Reply:
Good catch; I'll do that when I can switch back on this PR.

Reply:
Done.